src/services/__tests__/tourettes.test.js

```diff
@@ -18,31 +18,49 @@ describe('tourettes', () => {
   describe('sayTourettes', () => {
     describe('when input text is empty', () => {
       const inputText = '';
+      const minAmountOfReplies = 3;
 
       it('replies with at least three items from the profanities list', () => {
         const reply = sayTourettes(inputText);
+        const amountOfReplies = reply.split(' ').length;
         expect(reply).toNotBe(undefined);
-        expect(reply.split(' ').length).toBeGreaterThan(3);
+        expect(amountOfReplies).toBeGreaterThanOrEqualTo(minAmountOfReplies);
+      });
+    });
+
+    describe('when input text is integer', () => {
+      const amountOfProfanities = 25;
+      const inputText = `${amountOfProfanities}`;
+
+      it('replies with a specified amount of profanities', () => {
+        const reply = sayTourettes(inputText);
+        const amountOfReplies = reply.split(' ').length;
+        expect(reply).toNotBe(undefined);
+        expect(amountOfReplies).toBeGreaterThanOrEqualTo(amountOfProfanities);
       });
     });
 
     describe('when inputText is undefined', () => {
       const inputText = undefined;
+      const minAmountOfReplies = 3;
```

Comment:
On second thought though, we have other tests where `minAmountOfReplies` is 1, maybe let it be for the time being and we'll see about changing it later? Otherwise it's just gonna be littered with more or less descriptive constants I'm afraid.